functions/index.js

```diff
@@ -216,6 +217,27 @@ exports.listEncounters = functions.https.onRequest((request, response) => {
     });
 });
 
+exports.listEncountersByStatus = functions.https.onRequest((request, response) => {
```

Comment:
you might just add this functionality to listEncounters.  Seems like a reasonable option to have to the method in general.